docs/getting_started.rst

```diff
@@ -4,45 +4,82 @@ Getting started
 
 .. highlight:: bash
 
-Installation
-============
+**COMPAS FAB** can be easily installed on multiple platforms,
+using popular package managers such as conda or pip.
 
-The recommended way to install **COMPAS FAB** is to use `Anaconda/conda <https://docs.conda.io/>`_:
+Install with conda
+==================
+
+The recommended way to install **COMPAS FAB** is with `conda <https://conda.io/docs/>`_.
+For example, create an environment named ``my-project`` and install COMPAS and COMPAS FAB.
 
 ::
 
-    conda install -c conda-forge compas_fab
+    conda config --add channels conda-forge
+    conda create -n my-project compas_fab
 
+Afterwards, simply activate the environment
+and run the following command to check if the installation process was successful.
 
-But it can also be installed using ``pip``:
+.. code-block:: bash
 
-::
+    conda activate my-project
+    python -c "import compas_fab; print(compas_fab.__version__)"
+
+.. code-block:: none
+
+    0.17.0
```

Comment:
Should there be a hook to update this when the version is bumped?

Reply:
Oh, sorry, missed it.

Reply:
😉

Reply:
(this hook doesn't exist in compas)